virtual_ecosystem/core/constants.py

```diff
@@ -70,6 +70,20 @@ class CoreConsts(ConstantsDataclass):
     :cite:t:`fatichi_mechanistic_2019`. No empirical source is provided for this value.
     """
 
+    soil_moisture_capacity: float = 0.9
+    """Soil moisture capacity, unitless.
+
+    The soil moisture capacity, also known as field capacity or water holding capacity,
+    refers to the maximum amount of water that a soil can retain against the force of
+    gravity after it has been saturated and excess water has drained away. The value is
+    soil type specific, the format here is volumetric relative water content (unitless,
+    between 0 and 1).
+
+    TODO - This constant also exists in the hydrology model. There really should be a
```

Comment:
This is definitely something to talk about in more detail. We don't want such a key parameter in two places and potentially with different values. This relates to the metaconstants discussion in #364.

My hack solution to this is at the moment to load the constants from other models in setup, for example here: https://github.com/ImperialCollegeLondon/virtual_ecosystem/blob/2a33090e34121077767507d6f65718b0f40c51fc/virtual_ecosystem/models/abiotic/abiotic_model.py#L167

In the long-term this would ideally be a model input or could possibly even be calculated in `init`as a function of soil type? Let's discuss when we meet about #699

Reply:
I did try loading the constants from the hydrology model in setup initially, but the testing started to break (because the soil model gets used as an example for a lot of the core model. Decide to go with this approach rather than having to rewrite the tests in that section. I guess this might be an issue to resolve down the line